diagnostic.md

```diff
@@ -38,7 +52,12 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+$base-margin: 1.5em;
+$base-margin: 1.5em/2;
+
+.li {
+  margin: $base-margin;
```

Comment:
This should be `margin: $base-margin / 2;` since we're halving an existing variable's value.